src/engine/cachingreader/cachingreaderchunk.h

```diff
@@ -45,7 +53,7 @@ class CachingReaderChunk {
     CachingReaderChunk(const CachingReaderChunk&) = delete;
     CachingReaderChunk(CachingReaderChunk&&) = delete;
 
-    SINT getIndex() const {
+    SINT getIndex() const noexcept {
```

Comment:
We don't use `noexcept`anywhere. We might change this, but why introduce it in this unrelated PR?

Reply:
Are you asking me to remove all `noexcepts`?

Reply:
I didn't read all the coding style document.
When I write code I just write what I consider good C++ with the correct annotations, in order to gives semantic and improves compiler checks and code analysis.
I've a different approach to development than the mixxx team, so after this PR I'm out.

Reply:
Not necessarily, but please try to avoid such changes in the future unless you touch the line anyway. That would make the diff easier to review.

Reply:
> I didn't read all the coding style document.
> When I write code I just write what I consider good C++ with the correct annotations, in order to gives semantic and improves compiler checks and code analysis.

Sure, and that is fine. It's just easier to grasp the essence of the PR if the diff is small. 

> I've a different approach to development than the mixxx team, so after this PR I'm out.

I'm sorry to hear that, that was not my intention.

Reply:
It isn't about what you said specifically, but I didn't enjoy making this change in general. So I don't see myself going into a full refactoring of the audio buffers usage over all the app.

Reply:
> It isn't about what you said specifically, but I didn't enjoy making this change in general. 

Would you be interested in discussing your experience on Zulip? Maybe we can improve our DX, our workflows and our policies. Hopefully we can make contributing a more pleasant experience.